apps/main/src/llamalend/PageLlamaMarkets/cells/RateCell/RewardsIcons.tsx

```diff
@@ -1,30 +1,46 @@
+import uniq from 'lodash/uniq'
+import type { ReactElement } from 'react'
 import { LlamaMarket } from '@/llamalend/entities/llama-markets'
 import { useFilteredRewards } from '@/llamalend/PageLlamaMarkets/cells/cell.format'
 import type { RateType } from '@/llamalend/PageLlamaMarkets/hooks/useSnapshots'
 import Chip from '@mui/material/Chip'
 import Stack from '@mui/material/Stack'
-import { RewardIcons } from '@ui-kit/shared/ui/RewardIcon'
+import { RCCrvLogoMD } from '@ui/images'
+import { t } from '@ui-kit/lib/i18n'
+import { RewardIcon, RewardsImg } from '@ui-kit/shared/ui/RewardIcon'
 import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
 
-const { Spacing } = SizesAndSpaces
+const { IconSize } = SizesAndSpaces
+
+const RewardChip = ({ isLast, icon }: { isLast: boolean; icon: ReactElement }) => (
+  <Chip icon={icon} size="small" color="highlight" sx={{ ...(!isLast && { marginInline: '-8px' }) }} />
```

Comment:
Can get rid of the need for `isLast` by relying purely on CSS:
```suggestion
<Chip icon={icon} size="small" color="highlight" sx={{ '&:not(:last-child)': { marginInline: '-8px' } }} />
```